1.12.0/csi-operator-1.12.0-RC1.yaml

```diff
@@ -490,22 +490,6 @@ status:
   conditions: []
   storedVersions: []
 
----
```

Comment:
We need to remove `openebs-cstor-csi-iscsiadm` config map from this 1.12.0-RC1.yaml.

Reply:
Please refresh the page changes are already there ?